pkg/cluster/rook_ceph.go

```diff
@@ -480,21 +476,21 @@ func (c *Controller) SetFilesystemReplication(rookVersion semver.Version, name s
 		}
 		// Changing the size in the CephFilesystem has no effect so it needs to be set manually
 		// https://github.com/rook/rook/issues/3144
-		err := c.cephOSDPoolSetSize(rookVersion, RookCephSharedFSMetadataPool, level, cephcluster)
+		err := c.cephOSDPoolSetSize(rookVersion, cephVersion, RookCephSharedFSMetadataPool, level)
 		if err != nil {
-			return false, errors.Wrapf(err, "scale shared fs metadata pool size")
+			return true, errors.Wrapf(err, "scale shared fs metadata pool size")
```

Comment:
Hey @emosbaugh , why are we returning like it was "updated" now when in fact it was not ? I mean, doesn't this error mean that a failure happened therefore we haven't updated ? Trying to understand it.

Reply:
Good question

Reply:
Ya. I'm not so sure myself. I'll try to refactor the code a bit.

Reply:
I reverted this